cluster-autoscaler/core/utils.go

```diff
@@ -291,7 +308,7 @@ func getNodeInfoFromTemplate(nodeGroup cloudprovider.NodeGroup, daemonsets []*ap
 	pods = append(pods, baseNodeInfo.Pods()...)
 	fullNodeInfo := schedulernodeinfo.NewNodeInfo(pods...)
 	fullNodeInfo.SetNode(baseNodeInfo.Node())
-	sanitizedNodeInfo, typedErr := sanitizeNodeInfo(fullNodeInfo, id)
+	sanitizedNodeInfo, typedErr := sanitizeNodeInfo(fullNodeInfo, id, nil)
```

Comment:
Feels like we should also pass ignoredTaints here, no? You really shouldn't ignore a taint that is on your template, but if for some reason you do it would be inconsistent between scale-from-0 and normal scale-up.
I guess a possible use-case would be if your cloudprovider implementation automatically assumes some taint will be there, but in your cluster it won't, you may want to filter it out?

Reply:
My understanding is this call is used to sanitize the node info you get from the Node Group template (e.g. the one created from the taint tags on the ASG) for the scale-from-zero case. Presumably, you wouldn't have taints coming from that template you want to ignore.

That said, I don't mind adding it for consistency as it shouldn't hurt anything either.

Reply:
I didn't read your comment fully. I see that you already considered what I said. I'll make the change.